lib/parser/models.ex

```diff
@@ -0,0 +1,510 @@
+defmodule Thrift.Parser do
+
+  def parse(doc) do
```

Comment:
Document the `parse/1` and `parse/2`, and consider moving them to their own file?